src/v2/view-builder/views/TerminalView.js

```diff
@@ -50,8 +50,10 @@ const Body = BaseForm.extend({
   showMessages () {
     const messagesObjs = this.options.appState.get('messages');
     if (messagesObjs && Array.isArray(messagesObjs.value)) {
+      // To support intentional hard refresh in case of terminal states
+      // clear sessionStorage to initiate a new login flow
+      sessionStorage.removeItem('okta-siw-state-token');
```

Comment:
Looks like we can clear session storage independently with error messages.
Therefore we can override `postRender` of BaseView and clear up session there. (around line 85 below)

Reply:
as discussed, added to Terminal and Success Views..